doc/faq.md

```diff
@@ -0,0 +1,247 @@
+Irber et al., 2022. This is the fullest technical description of FracMinHash available.
+
+[Mash: fast genome and metagenome distance estimation using MinHash](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0997-x),
+Ondov et al., 2016. This is the original paper that inspired
+sourmash. It discusses sketching with MinHash and does a great job of
+showing how well Jaccard estimation works for comparing genomes! A
+good contrasting point to take into account is that _MinHash cannot do
+overlap or containment estimation_, which nicely motivates the
+previous paper and the next two.
+
+[Mash Screen: high-throughput sequence containment estimation for genome discovery](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1841-x),
+Ondov et al., 2019; and
+[CMash: fast, multi-resolution estimation of k-mer-based Jaccard and containment indices](https://academic.oup.com/bioinformatics/article/38/Supplement_1/i28/6617499). Both
+papers discusses containment and metagenome analysis extensively, and
+use an approach that can be usefully contrasted with sourmash. There
+is a nice blog post on
+[mash screen](https://genomeinformatics.github.io/mash-screen/) that
+is worth reading, too!
+
+If you want a nice chaser, please see this section at the end of the
+blog post above:
+
+>It would be great to see additional methods developed to process
+containment scores, reduce the output redundancy, and report accurate
+compositional estimates for metagenomes. One easy approach is a
+“winner take all” model, like sourmash implements.
+
+## What k-mer size(s) should I use with sourmash?
+
+The short answer is: for DNA, use k=31.
+
+Slightly longer answer: when we look at the k-mer distribution
+across all of the bacterial genomes in GTDB, we find that 99% or
+more of 31-mers are _genome_, _species_ or _genus_ specific.
+
+If you go lower (say, k=21), then you get a few percent of k-mers
+that match above the genus level - family or above.
+
+If you go higher (k=51), a higher percentage of k-mers are genome-specific.
+
+For the core sourmash operations - search, gather, and compare - we
+believe (with evidence!) that (a) the differences between k=21, k=31,
+and k=51 are negligible; and that (b) k=31 works fine for most
+day-to-day use of sourmash.
+
+We also provide [Genbank and GTDB databases](databases.md) for k=21,
+k=31, and k=51.
+
+For some background on k-mer specificity, we recommend this paper:
+[MetaPalette: a k-mer Painting Approach for Metagenomic Taxonomic Profiling and Quantification of Novel Strain Variation](https://journals.asm.org/doi/10.1128/msystems.00020-16),
+Koslicki & Falush, 2016.
+
+## How do k-mer-based analyses compare with read mapping?
+
+tl;dr very well! But it's a bit one sided: if k-mers match, reads will
+map, but not necessarily vice versa. So read mapping rates are almost always
```

Comment:
What is the explicit 'vice versa'? 'if reads map, kmers will match?' What does that mean?
I guess I just don't understand the fundementals of this section, may be worth adding a longer section before the tldr.

Reply:
exactly: if reads map, k-mers _may not_ match. Will think on how and if to improve text!